### PR TITLE
Prepare for release v1.16.0-rc.0

### PR DESCRIPTION
**Pull request by aanm**
Prepare for release v1.16.0-rc.0

See the included CHANGELOG.md for a full list of changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+* Introduce --force-device-detection option to apply the auto-detection criteria also when devices are explicitly listed with --devices. (cilium/cilium#32730, @kvaps)
+* Introduce granular etcd permissions to access KVstoreMesh cached data (cilium/cilium#33082, @giorio94)
+* More validation has been added to the CiliumNetworkPolicy and CiliumClusterwideNetworkPolicy CRDs. Policies that may have been ignored by the Cilium agent will now be rejected by the Kubernetes API server. (cilium/cilium#32814, @squeed)
+* Remove etcd.managed Helm setting (cilium/cilium#32921, @joestringer)
+* Support Egress Gateway for endpoints that are also selected by a L7 Network Policy. (cilium/cilium#32828, @ysksuzuki)
+* Switch the RBAC used for hubble certificate generation in `cronJob` mode to namespace-scoped. (cilium/cilium#33027, @giorio94)
+* ui: v0.13.1 release (cilium/cilium#32852, @geakstr)
+* When upgrading, users can experience a change to their configuration if they were overriding the `k8s-heartbeat-timeout` flag.
 K8s client timeout and keep alive are no longer getting values from the `k8s-heartbeat-timeout` flag, but have default values (30 seconds). (cilium/cilium#32625, @dlapcevic)
```
Comment (qmonnet) [inline CHANGELOG.md]: Nit: Could we remove this `^M` character here?

**Review by qmonnet — APPROVED** [review]
Seems all in order, thank you!

One non-blocking nit below

File: CHANGELOG.md (hunk 1 above)
Comment (aanm) [reply CHANGELOG.md]: Fixed

**Pull request merged**